test/integration/mon/inspec/mon_spec.rb

```diff
@@ -1,9 +1,5 @@
-require 'serverspec'
-
-set :backend, :exec
-
 describe file('/etc/munin/plugin-conf.d/nginx') do
```

Comment:
Let's switch this to the ini resource.

Reply:
ini isn't exactly working. It seems to expect files to be formatted `key = value`. parse_config_file isn't a good candidate either because there's no way to format the assignment regex to handle a '.' in the key value.

Reply:
@Vancelot11 look at how the [ini resource](https://www.inspec.io/docs/reference/resources/ini/) documentation describes "sections".